falcon/api.py

```diff
@@ -364,6 +364,8 @@ def add_static_route(self, prefix, directory, downloadable=False):
             Serving files directly from the web server,
             rather than through the Python app, will always be more efficient,
             and therefore should be preferred in production deployments.
+            For security reasons, the prefix forder and the fallback_filename (if provided)
+            should be read only for the account running the application.
```

Comment:
spelling nit: I believe `forder` should be `order`

Reply:
There is an error, instead of prefix I should have used directory